version: "3"

services:
  localstack:
    image: localstack/localstack:0.11.2 # TODO Atualizar Versao, problema na na versao 0.12.16 ao conectar no dynamo
    ports:
      - "4566-4599:4566-4599"
      - "8085:8080"
    environment:
      - SERVICES=dynamodb,s3
    #volumes:
      #- './.dockervolume/.localstack:/tmp/localstack'
      #- '/var/run/docker.sock:/var/run/docker.sock'
  redis:
    image: redis
    container_name: redis
    ports:
      - 6379:6379
    volumes:
        - './.dockervolume/.redis:/redis'
  redisinsight:
    image: redislabs/redisinsight:latest
    ports:
      - '8001:8001'
    #volumes:
      #- './.dockervolume/.redis:/db'